doc/hardware/peripherals/mipi.rst

```diff
@@ -0,0 +1,13 @@
+.. _mipi_api:
+
+MIPI
+####
+
+Overview
+********
+
```

Comment:
if overview is empty, just delete it or add some introduction.

Reply:
Deleted